scripts/globals/shop.lua

```diff
@@ -93,6 +93,30 @@ dsp.shop =
         dsp.shop.general(player, stock)
     end,
 
+    --[[ *******************************************************************************
+        send outpost shop dialog to player
```

Comment:
"outpost"?

Reply:
Sorry I forgot to fix the comment @TeoTwawki , don't hit me too hard lol

Reply:
_-threatens with a stick-_

Reply:
fixed. Resolving conversation.